src/Akka.Streams.Kafka/Stages/ConsumerStage.cs

```diff
@@ -102,6 +111,7 @@ public override void PreStart()
             _messagesReceived = GetAsyncCallback<Message<K, V>>(MessagesReceived);
             _partitionsAssigned = GetAsyncCallback<IEnumerable<TopicPartition>>(PartitionsAssigned);
             _partitionsRevoked = GetAsyncCallback<IEnumerable<TopicPartition>>(PartitionsRevoked);
+            ScheduleRepeatedly(TimerKey, _settings.PollInterval);
```

Comment:
We can't use `ScheduleOnce` with Confluent.Kafka, because if we stop polling the Kafka we just drop a connection, and we won't get any errors. We should poll it without any pauses